recipes/_install.rb

```diff
@@ -29,16 +29,16 @@ def nsp
 include_recipe 'cerner_splunk::_restart_marker'
 
 # Actions
-# This service definition is used only for ensuring splunk is started during the run
-service 'splunk-start' do
+# This service definition is used for ensuring splunk is started during the run and to stop splunk service
+service 'splunk' do
   service_name service
   action :nothing
-  supports status: true, start: true
+  supports status: true, start: true, stop: true
   notifies :delete, 'file[splunk-marker]', :immediately
```

Comment:
That was my initial thought. However I didn't feel good about making 'splunk-start' support 'stop' since <code>notifies :stop, 'service[splunk-start]'</code> looks a little confusing and hence went with 'splunk-stop'. We can go that way if everyone feels so.